lib/fileFactory.js

```diff
@@ -50,7 +50,7 @@ module.exports = (options, fileUploadOptions = {}) => {
     tempFilePath: options.tempFilePath,
     truncated: options.truncated,
     mimetype: options.mimetype,
-    md5: options.hash,
+    [fileUploadOptions.hashAlgorithm || 'md5']: options.hash,
```

Comment:
couldn't we just use `hash` here as the object key?

Reply:
yes, but that a breaking change require a major version bump.
doing its this way make it backward compatiable with existing code expecting the `md5` field